src/Sarif.UnitTests/Visitors/MakeUriAbsoluteVisitorTest.cs

```diff
@@ -11,26 +11,25 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
     public class MakeUriAbsoluteVisitorTest
     {
-        private Run GenerateRunForTest(Dictionary<string, Uri> uriBaseIdMapping)
+        private Run GenerateRunForTest(Dictionary<string, FileLocation> uriBaseIdMapping)
         {
             Run run = new Run();
             run.Files = new List<FileData>(new[]
             {
                 new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/file1.cs", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 0 } },
                 new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/file2.dll", UriKind.Relative), UriBaseId="%TEST2%", FileIndex = 1 } },
                 new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/archive.zip", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 2 } },
-                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/archive.zip#file3.cs", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 3 } },
-                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/archive.zip#archive2.gz", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 4 } },
-                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/archive.zip#archive2.gz/file4.cs", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 5 }, },
-                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("src/archive.zip#file5.cs", UriKind.Relative), UriBaseId="%TEST1%", FileIndex = 6 },  }
+                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("file3.cs", UriKind.Relative), FileIndex = 3 }, ParentIndex = 2 },
+                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("archive2.gz", UriKind.Relative), FileIndex = 4 }, ParentIndex = 2 },
+                new FileData() { FileLocation=new FileLocation{ Uri=new Uri("file4.cs", UriKind.Relative), FileIndex = 5 }, ParentIndex = 4 },
             });
 
             if (uriBaseIdMapping != null)
             {
                 run.Properties = new Dictionary<string, SerializedPropertyInfo>();
-
-                run.Properties[RebaseUriVisitor.BaseUriDictionaryName] = RebaseUriVisitor.ReserializePropertyDictionary(uriBaseIdMapping);
             }
+
```

Comment:
Not necessary, removed.

Reply:
Fixed.

---
In reply to: [247331466](https://github.com/Microsoft/sarif-sdk/pull/1210#discussion_r247331466) [](ancestors = 247331466)